key_management.md

```diff
@@ -5,22 +5,22 @@ css_id: key_management
+On both the Director and the Image repository, the OEM maintains the keys to the Root, Timestamp, Snapshot, and Targets roles. However, for any delegated Targets roles on the Image repository, the corresponding keys are expected to be maintained by the supplier to which they have been delegated. For example, if a tier-1 supplier signs its own images, then the supplier would maintains its offline keys. 
 
 
 ### Online vs. offline keys
 
 Repository administrators SHOULD use offline keys to sign the Root metadata on the Director repository, so attackers cannot tamper with this file after a repository compromise. The Timestamp, Snapshot, and Targets metadata SHOULD be signed using online keys, so that an automated process can instantly generate fresh metadata.
 
-On the Image repository, the Timestamp and Snapshot metadata SHOULD be signed using online keys, so that an automated process can instantly generate fresh metadata. There are two options for signing the Timestamp and Snapshot metadata, each with the opposite trade-off from the other. In the first option, the OEM uses online keys meaning automated processes can automatically renew the Timestamp and Snapshot metadata to indicate that there are new Targets metadata and/or images available. However, attackers who have compromised a supplier's key would now have access to the Image repository and can instantly publish malicious images. If they additionally compromise the Director repository, then they can execute arbitrary software attacks by selecting these malicious images on the Image repository for installation. Such an attack could also facilitate mix-and-match attacks.
+On the Image repository, the Timestamp and Snapshot metadata SHOULD be signed using online keys, so that an automated process can instantly generate fresh metadata. There are two options for signing the Timestamp and Snapshot metadata, each with the opposite trade-off from the other. In the first option, the OEM uses online keys, meaning automated processes can automatically renew the Timestamp and Snapshot metadata to indicate that new Targets metadata and/or images are available. However, in this scenario, attackers who have compromised a supplier's key would now have access to the Image repository and can instantly publish malicious images. If these attackers also compromise the Director repository, then they can execute arbitrary software attacks by selecting these malicious images on the Image repository for installation. Such an attack could also facilitate mix-and-match attacks.
```

Comment:
```suggestion
On the Image repository, the Timestamp and Snapshot metadata SHOULD be signed using online keys, so that an automated process can instantly generate fresh metadata. There are two options for signing the Timestamp and Snapshot metadata, each with the opposite trade-off from the other. In the first option, the OEM uses online keys, meaning automated processes can automatically renew the Timestamp and Snapshot metadata to indicate that new Targets metadata and/or images are available. However, in this scenario, attackers who have compromised a supplier's key as well as the Image repository could now instantly publish malicious images. If these attackers also compromise the Director repository, then they can execute arbitrary software attacks by selecting these malicious images on the Image repository for installation. Such an attack could also facilitate mix-and-match attacks.
```